apps/taxes/src/modules/trpc/trpc-app-router.ts

```diff
@@ -23,6 +25,8 @@ export const appRouter = router({
   taxJarTaxCodes: taxJarTaxCodesRouter,
   taxJarMatches: taxJarTaxCodeMatchesRouter,
   avataxMatches: avataxTaxCodeMatchesRouter,
+  avataxClientLogs: avataxClientLoggerRouter,
+  taxjarClientLogs: taxjarClientLoggerRouter,
```

Comment:
I dont see a reason to have duplicated routers

you can have single router and use query/input to decide which config is log assigned to

eg in CMS app there is single "CRUD" procedure for every instance/configuration. They just have "type" field